src/app/side-bar/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "src/assets/styles/variables";
+
+.dashboard-container {
+  width: auto;
+  background-color: floralwhite;
```

Comment:
please don,t use the color name for the value.
Please make variable and try to use the variable instead.